progressist/__init__.py

```diff
@@ -115,12 +115,28 @@ def speed(self):
         """Number of iterations per second."""
         return Float(1.0 / self.avg if self.avg else 0)
 
+    @property
+    def throttled(self):
+        if not self.throttle:
+            return
```

Comment:
I know it's equivalent, but since this property is expected to return a boolean... Can you make it return an actual `False` here instead of `None`?

I know, `bool(None)` is `False`, but... I've been burnt in the past by APIs where you wanted to distinguished between `None` and `False`, and since then I prefer being explicit. :smiley: